### ISSUE TITLE
spkl: PackageReference in csproj

### ISSUE THREAD

**Issue by derekfinlinson** [scottdurow/SparkleXrm#274]
Using the new PackageReference in your .csproj file instead of a packages.config file doesn't work. It doesn't copy the content files into your project. Based on [new documentation](https://docs.microsoft.com/en-us/nuget/reference/nuspec#including-content-files), it's due to some changes in the nuspec structure. If you can't get to this soon, I'd be happy to do a pull request for it. Thanks!

**Comment by scottdurow** [comment]
Thank you for spotting this. I would really appreciate a PR! Thank you 😊

**Comment by scottdurow** [comment]
Hi @derekfinlinson - would you be able to document the change needed or submit a PR? Thanks!

**Comment by derekfinlinson** [comment]
I was working on it but can't figure out how to get the content files to copy to the project folder. I thought using `<contentFiles>` instead would work but it doesn't seem to. There is an [issue](https://github.com/NuGet/docs.microsoft.com-nuget/issues/775) open in the Nuget repo to add better documentation to accomplish this but I haven't had a chance to dig into it yet. Sounds like it's possible just need to figure out how.

**Comment by Sano** [comment]
You need to modify SPKL to look in {user profile directory}/.nuget/packages.

**Comment by derekfinlinson** [comment]
I don't think that will work since we need the spkl files included in each individual project.

**Comment by MrCliffster** [comment]
For anyone searching - a new project in VS2017 or VS2019 will use PackageReferences by default, and this issue shows that you need to revert to using packages.config to enable spkl to generate the right files.
